### ISSUE TITLE
"Incorrect number of AD fields (2) at <chrom:pos>, cannot merge."

### ISSUE THREAD

**Issue by katherinesmith** [Illumina/agg#6]
I'm encountering a problem at the ingestion 2 step using agg c8173c2. Attempts to build chunks for several batches of 500 samples fail for all batches with the error message above.

One job reports the problem as being 1:815520. Narrowing the problem down to a batch of 10 samples reveals that two have non-printing characters in the .bcf produced by the ingest1 step at this location:

```
> bcftools view -H -r 1:815520-815520 ingest1/A.bcf
1	815520	.	C	T��	2577	.	.	GT:GQ:GQX:DPI:AD:PF	0/1:2054:2054:123:50,63:0
1	815520	rs61768238	C	T	158	.	.	GT:GQ:GQX:DP:DPF:AD:PF	0/1:191:35:52:1:33,19:0

> bcftools view -H -r 1:815520-815520 ingest1/B.bcf
1	815520	.	C	T,�	2266	.	.	GT:GQ:GQX:DPI:AD:PF	0/1:2306:2266:119:58,59:1```
```
The original gVCF entries are:
```
bcftools view -H -r 1:815520-815520 A.genome.vcf.gz
1	815519	.	CCTTTATTC	CTTTAATTA	2577	HighDepth	CIGAR=1M8D8I;RU=.;REFREP=.;IDREP=.;CSQT=|ENST00000594233|upstream_gene_varian	GT:GQ:GQX:DPI:AD	0/1:2054:2054:123:50,63
1	815520	rs61768238	C	T	158	SiteConflict;HighDepth	SNVSB=-17.2;SNVHPOL=4;CSQT=|ENST00000594233|upstream_gene_variant	GT:GQ:GQX:DP:DPF:AD	0/1:191:35:52:1:33,19
```

and

```
bcftools view -H -r 1:815520-815520 B.genome.vcf.gz
1	815519	.	CCTTTATTC	CTTTAATTA	2266	PASS	CIGAR=1M8D8I;RU=.;REFREP=.;IDREP=.;CSQT=|ENST00000594233|upstream_gene_variant	GT:GQ:GQX:DPI:AD	0/1:2306:2266:119:58,59
1	815520	rs61768238	C	T	101	SiteConflict	SNVSB=-3.7;SNVHPOL=4;CSQT=|ENST00000594233|upstream_gene_variant	GT:GQ:GQX:DP:DPF:AD	0/1:134:29:56:3:41,15
```

It looks like the problem could be the step that splits MNVs into SNVs.

**Comment by jaredo** [comment]
This definitely looks like a bug. Will investigate.

**Comment by jaredo** [comment]
Fixed in 29b2595d4ed3d3a863039b15e5aee5a0dd718727
